frozenlist/__init__.py

```diff
@@ -73,7 +79,43 @@ def __hash__(self):
         else:
             raise RuntimeError("Cannot hash unfrozen list.")
 
+    def __deepcopy__(self, memo: dict[int, Any]):
```

Comment:
Use `object` whenever possible, avoid `Any`.
```suggestion
    def __deepcopy__(self, memo: dict[int, object]):
```